apypie/api.py

```diff
@@ -176,11 +176,13 @@ def http_call(self, http_method, path, params=None, headers=None, options=None):
         if headers:
             kwargs['headers'] = headers
 
-        if params:
-            if http_method == 'get':
+        if http_method in ['get', 'head']:
```

Comment:
could this be 
```suggestion
        if http_method in ['get', 'head'] and params:
```

Reply:
...and dropping the `if` on the next line? That should be equivalent. If you prefer so.

Reply:
I refactored it to have on less conditional test.